### PR TITLE
add support for installing on mac osx

### PR DESCRIPTION
**Pull request by dgriff-tools**

### CODE REVIEW

**Comment by molner** [comment]
LGTM!

File: install.sh [hunk 1]
```diff
@@ -188,14 +195,19 @@ parse_args_wrapper() {
...
 
 determine_os() {
-    #determine hostOS for newer versions of Linux
-    hostOS=$(cat /etc/*-release | grep PRETTY_NAME | grep -o '".*"' | sed 's/"//g' | sed -e 's/([^()]*)//g' | sed -e 's/[[:space:]]*$//')
-    if [ ! -f /etc/redhat-release ]
-        then
-        hostOS_2=null_os
+
+    if [ "$(uname)" == "Darwin" ]; then
+        hostOS="Mac OS X"
```
Comment (tedoc2000) [inline install.sh:200]: Lets do it with no spaces.. Spaces always cause problems somewhere.

File: install.sh [hunk 2]
```diff
@@ -234,6 +246,10 @@ assign_needed_os() {
         8)
             hostOS="Debian GNU/Linux 8"
         ;;
+        #Mac OS X
+        9)
+            hostOS="Mac OS X"
```
Comment (tedoc2000) [inline install.sh]: No spaces.

File: install.sh (hunk 1 above)
Comment (dgriff-tools) [inline install.sh:200]: I'll make that change

**Comment by dgriff-tools** [comment]
I'll make that change

File: install.sh (hunk 1 above)
Comment (dgriff-tools) [inline install.sh:200]: done

File: install.sh (hunk 2 above)
Comment (dgriff-tools) [inline install.sh]: done

**Pull request merged**